libs/plugin/src/generators/convert-host-binding/generator.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { Tree } from '@nx/devkit';
+import { createTreeWithEmptyWorkspace } from '@nx/devkit/testing';
+
+import { convertHostBindingGenerator } from './generator';
+import { ConvertHostBindingGeneratorSchema } from './schema';
+
+const filesMap = {
```

Comment:
I'd like to see some more test cases. And before we merge this one, we will need to make sure that we don't break apps.

Reply:
Added more test cases, additionally tested in our organisation's repo with forked library version

Reply:
I also fixed some minor bugs identified by the additional tests I implemented.